sdk/servicebus/Azure.ResourceManager.ServiceBus/api/Azure.ResourceManager.ServiceBus.net8.0.cs

```diff
@@ -844,6 +932,212 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.ServiceBus.Models.Mode left, Azure.ResourceManager.ServiceBus.Models.Mode right) { throw null; }
+        public static implicit operator Azure.ResourceManager.ServiceBus.Models.Mode (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.ServiceBus.Models.Mode left, Azure.ResourceManager.ServiceBus.Models.Mode right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class NamespaceReplicaLocation : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>
+    {
+        public NamespaceReplicaLocation() { }
+        public string ClusterArmId { get { throw null; } set { } }
+        public string LocationName { get { throw null; } set { } }
+        public Azure.ResourceManager.ServiceBus.Models.GeoDRRoleType? RoleType { get { throw null; } set { } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceBus.Models.NamespaceReplicaLocation>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class NetworkSecurityPerimeter : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ServiceBus.Models.NetworkSecurityPerimeter>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ServiceBus.Models.NetworkSecurityPerimeter>
```

Comment:
this name is quite generic and might collide with other RP's types.
Can we rename this to `ServiceBusNetworkSecurityPerimeter`?